### DISCUSSION TITLE
Feature request, seeing another person's profile

### DISCUSSION THREAD

**Discussion by vycdev** [monkeytypegame/monkeytype#1480] [Ideas] [answered]
Hello, here's a request, would be nice to be able to see another person's profile if you know their nickname.
A derived idea from this, to be able to add friends?
Anyway what do you guys think?

**Comment by Miodec** [comment, marked as answer]
Yeah, already planned.

**Reply by vycdev** [reply]
Thank you

**Reply by ShizukoV** [reply]
When will it come @Miodec

**Reply by unrealapex** [reply]
#563 and #1038
